patternfly/patternfly-elements
# feat(tile): PF1:1

### What I did

1. Added BaseTile class to inherit from
2. Added `selected` attribute to highlight the tile
3. Added `stacked` attribute which has two sizes `md` and `lg` to align the icon above the title

### Testing Instructions
Load up the deploy preview for this branch
Load up [PatternFly v4 Tabs](https://www.patternfly.org/v4/components/tile/)
Compare and contrast
Read over the [docs](https://deploy-preview-2187--patternfly-elements.netlify.app/components/tile/) and see if anything's missing
